### ISSUE TITLE
Code Browser window — statusbar is completely empty

### ISSUE THREAD

**Issue by furious-programming** [zamtmn/metadarkstyle#14]
Statusbar in the `Code Browser` window is empty:

![image](https://user-images.githubusercontent.com/8170730/233868624-e51d4c82-c386-4fc3-9bd0-ea622be02e24.png)

**Comment by klaus101** [comment]
Without looking at the Code Browser itself -> kidz noticef occasionally that the dark theme for the StatusBar obviously does not work if the  StatusBar panel is a **SimplePanel**  (maybe that's with the Code Browser?)

That needs to be fixed in the StatusBar painting routine . ==> it hould cover **SimplePanel** too.

**Comment by klaus101** [comment]
Probably by something like such (i tried it, works):  in uwin32widgetsetdark.pas:

```
function StatusBarWndProc(Window: HWND; Msg: UINT; wParam: Windows.WPARAM; lParam: Windows.LPARAM; uISubClass: UINT_PTR; dwRefData: DWORD_PTR): LRESULT; stdcall;
// . . . 
      LCanvas.Pen.Color:= SysColor[{COLOR_BTNFACE}COLOR_GRAYTEXT];
      if StatusBar.SimplePanel then
         LCanvas.TextOut(X+3, (StatusBar.Height - LCanvas.TextHeight('Ag')) div 2, StatusBar.SimpleText)
      else
      for Index:= 0 to StatusBar.Panels.Count - 1 do
// . . . 
```
zamtmn, could you please check if it makes sense to you?

**Comment by zamtmn** [comment]
Thank you!

**Comment by furious-programming** [comment]
Wait, something is wrong. I downloaded the fresh sources of this package and installed, and the IDE is white... I completely uninstalled the metadarkstyle package (both packages), then again installed and still the IDE is white. The sources from before changes works, fresh not. Do you know why?

**Comment by zamtmn** [comment]
I didn't notice any problems. what is your previous version? these changes do not affect the work in general, the problem is something else

**Comment by furious-programming** [comment]
Yes, I know that it should not affect the work in general but, the problem described is 100% replicable. I tried fow times and every time the IDE is white. The version that works for me is the one from two weeks ago, but I don't know exactly because I just downloaded the sources and installed them (I didn't create a repo clone or something).

**Comment by klaus101** [comment]
Did you do an comprehensive compare of both packages if the few lines in speech (if StatusBar.SimplePanel then ... else) are really the only difference?
You could comment out those lines for to check. It should be a very very crazy dependency if they should cause it.

**Comment by zamtmn** [comment]
>>The version that works for me is the one from two weeks ago
please use git, and find the comit that introduces the problem
you probably missed the moment when the settings appeared:
![image](https://user-images.githubusercontent.com/39580611/235449591-11551115-a9df-4d5e-a054-cda7005d9362.png)

**Comment by furious-programming** [comment]
Damn, I missed it. This is I suggested to write `readme.md` and describe what features are implemented, how to install it and how to used it properly. I installed fresh package again and the IDE actually is white by default, but when I change the settings (in the IDE options window, as you pointed) and restart the IDE, it is dark, as expected. In summary, it works fine, there are no problems.
